lib/dialog/style.scss

```diff
@@ -39,6 +39,7 @@
   display: flex;
   flex-direction: column;
   flex: 1 0 auto;
+  max-height: calc(100vh - 56px - 20px);
```

Comment:
This change comes from https://github.com/Automattic/simplenote-electron/pull/2815.